### PR TITLE
Document crd-install annotation for cluster registry crd.

### PR DESCRIPTION
**Pull request by gyliu513**
/sig multicluster

/cc @font @madhusudancs @perotinus @pmorie

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -58,6 +58,16 @@ $ chown -R $USER docs/reference/build  # The generated docs are owned by root.
 $ kubebuilder create config --crds --output cluster-registry-crd.yaml
 ```
 
+**NOTE:** If you want to use `cluster-registry-crd.yaml` in a helm chart, then it is
+suggested to add the following annotation to `cluster-registry-crd.yaml`, this can
```
Comment (perotinus) [inline docs/development.md]: Can you update the text a bit?

"... `cluster-registry-crd.yaml`. This ensures that the cluster registry CRD is created before other resources in the Helm chart. This annotation is available in Helm 2.10+."

Comment (gyliu513) [reply docs/development.md]: Done. Thanks @perotinus

**Comment by perotinus** [comment]
@gyliu513 It looks like the test actually failed this time. Can you investigate?

**Pull request merged**